core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -180,14 +187,19 @@ export class DotContainerPropertiesComponent implements OnInit {
     clearContent(): void {
         this.dotAlertConfirmService.confirm({
             accept: () => {
-                this.store.updateContentTypeVisibility(false);
-                this.form.reset();
                 this.form.get('containerStructures').clearValidators();
                 this.form.get('containerStructures').reset();
+                this.form.get('preLoop').reset();
+                this.form.get('postLoop').reset();
                 // clear containerStructures array
                 (this.form.get('containerStructures') as FormArray).clear();
                 this.form.get('code').addValidators(Validators.required);
                 this.form.updateValueAndValidity();
```

Comment:
Can you move this to a private method and add a name that can be more explicit in what we do.

Reply:
Noted

Reply:
updated